src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -241,7 +279,30 @@ def check_read_nwb(self, nwbfile_path: str):
                 if recording.has_scaled_traces() and nwb_recording.has_scaled_traces():
                     check_recordings_equal(RX1=recording, RX2=nwb_recording, return_scaled=True)
 
-    def check_align_starting_time_internal(self):
+    def check_interface_align_timestamps(self):
+        fresh_interface = self.data_interface_cls(**self.test_kwargs)
+        if fresh_interface._number_of_segments == 1:
+            unaligned_timestamps = self.interface.get_timestamps()
+            aligned_timestamps = unaligned_timestamps + 1.23 + np.random.random(size=unaligned_timestamps.shape)
```

Comment:
non-deterministic, also same question, how comes this is not inherited from the mixing?

Reply:
Fixed